### PR TITLE
feat: improve logging in handler and broadcaster

### PR DESCRIPTION
**Pull request by sammy1991106**
## Todos

- [ ] Unit tests
- [ ] Manual tests
- [ ] Documentation
- [ ] Connect epics/issues
- [ ] Tag type of change

### CODE REVIEW

File: src/log.rs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+use std::fmt::Display;
+
+use valuable::Valuable;
+
+#[derive(Valuable)]
+pub struct LoggableVec(Vec<String>);
```
Comment (cgorenflo) [inline src/log.rs:6]: Is `Valuable` not derived for `Vec<String>`?

File: src/handlers/evm_verify_msg.rs [hunk 2]
```diff
@@ -185,6 +189,19 @@ where
...
+        let message_ids = messages
+            .iter()
+            .map(|message| format!("{}{}{}", message.tx_id, ID_SEPARATOR, message.log_index))
+            .collect::<Vec<_>>();
+        info!(
+            poll_id = poll_id_str,
+            source_chain = source_chain_str,
+            message_ids = LoggableVec::new(&message_ids).as_value(),
+            "start verifying messages in poll",
+        );
```
Comment (cgorenflo) [inline src/handlers/evm_verify_msg.rs:203]: you could make use of a span here so you don't have to repeat the attached values for both info logs

File: src/log.rs (hunk 1 above)
Comment (sammy1991106) [reply src/log.rs:6]: ah, it is! nvm I'll get rid of this. silly

File: src/handlers/evm_verify_msg.rs (hunk 2 above)
Comment (sammy1991106) [reply src/handlers/evm_verify_msg.rs:203]: will try

**Review by cgorenflo — APPROVED** [review]

**Pull request merged**